### ISSUE TITLE
Custom Tooltip for Line Charts Produces error

### ISSUE THREAD

**Issue by nathanloehlein** [plouc/nivo#278]
If I follow the code for the line chart with custom tooltip

```tooltip={slice => {
            return (
                <div>
                    <h2>{slice.id}</h2>
                    {slice.points.map((e, i) => (
                        <p key={i}>
                            <strong>{e.id}:</strong> {e.value}
                        </p>
                    ))}
                </div>
            )
        }}
```

error: ` Warning: Failed prop type: Invalid prop `tooltip` of type `function` supplied to `LineSlicesItem`, expected a single ReactElement.`

https://github.com/plouc/nivo/blob/4aeed5d87159164df0f045a125131f29bc2e2e5a/packages/line/src/LineSlices.js#L57 and https://github.com/plouc/nivo/blob/4aeed5d87159164df0f045a125131f29bc2e2e5a/packages/line/src/LineSlicesItem.js#L61 not sure if these should take a func or if something is wrong with the code.

**Comment by plouc** [comment]
Fixed by #275

**Comment by nathanloehlein** [comment]
Thank you! Sorry I didn't see the previous issue.
